powershell-adapter/psDscAdapter/psDscAdapter.psm1

```diff
@@ -469,14 +471,18 @@ function Invoke-DscOperation {
 
                     switch ($Operation) {
                         'Get' {
-                            $Result = $dscResourceInstance.Get()
+                            $Result = @{}
+                            $raw_obj = $dscResourceInstance.Get()
+                            $ValidProperties | %{ $Result[$_] = $raw_obj.$_ }
```

Comment:
Best practice in scripts is to not use alias, replace all use of `%` with `ForEach-Object`

Reply:
Updated.